Release notes v0.36 — qutrits (#5606)
PennyLaneAI/pennylane: doc/releases/changelog-0.36.0.md

  device (shortname: `"default.qutrit.mixed"`).
  [(#5495)](https://github.com/PennyLaneAI/pennylane/pull/5495)
  [(#5451)](https://github.com/PennyLaneAI/pennylane/pull/5451)
  [(#5186)](https://github.com/PennyLaneAI/pennylane/pull/5186)
  [(#5082)](https://github.com/PennyLaneAI/pennylane/pull/5082)
  [(#5213)](https://github.com/PennyLaneAI/pennylane/pull/5213)

  Mixed-qutrit circuits have access to the same operations, measurements, and 
  functionality as circuits programmed on `DefaultQutrit`.

  ```python
  dev = qml.device("default.qutrit.mixed", wires=1)

  def circuit():
      qml.TRY(0.1, wires=0)

  @qml.qnode(dev)
  def shots_circuit():
      circuit()
      return qml.sample(), qml.expval(qml.GellMann(wires=0, index=1))

  @qml.qnode(dev)
  def density_matrix_circuit():
      circuit()
      return qml.state()
  ```

  ```pycon
  >>> shots_circuit(shots=5)
  (array([0, 0, 0, 0, 0]), 0.19999999999999996)
  >>> density_matrix_circuit()
  tensor([[0.99750208+0.j, 0.04991671+0.j, 0.        +0.j],
        [0.04991671+0.j, 0.00249792+0.j, 0.        +0.j],
        [0.        +0.j, 0.        +0.j, 0.        +0.j]], requires_grad=True)
  ```

>   Keep your eyes peeled for qutrit noise model support in the coming releases!

Author: This'll probably be fine?

> <h4>Simulate mixed-state qutrit systems 3️⃣</h4>

Reviewer: ```suggestion
<h3>Improvements 🛠</h3>

<h4>Simulate mixed-state qutrit systems</h4>
```
Let's make this part of "improvements" since there are follow up part of this coming in a future release.

> * Mixed qutrit states can now be simulated with the `"default.qutrit.mixed" device.

Reviewer: ```suggestion
* Mixed qutrit states can now be simulated with the `"default.qutrit.mixed"` device.
```

  [(#5495)](https://github.com/PennyLaneAI/pennylane/pull/5495)
  [(#5451)](https://github.com/PennyLaneAI/pennylane/pull/5451)
  [(#5186)](https://github.com/PennyLaneAI/pennylane/pull/5186)
  [(#5082)](https://github.com/PennyLaneAI/pennylane/pull/5082)
  [(#5213)](https://github.com/PennyLaneAI/pennylane/pull/5213)

  Thanks to contributors from the University of British Columbia, a mixed-state
  qutrit device is now available for simulation, providing a noise-capable

>   equivalent to `DefaultQutrit`.

Reviewer: ```suggestion
  equivalent to `"default.qutrit"`.
```

  ```python
  dev = qml.device("default.qutrit.mixed")

  def circuit():
      qml.TRY(0.1, wires=0)

  @qml.qnode(dev)
  def shots_circuit():
      circuit()
      return qml.sample(), qml.expval(qml.GellMann(wires=0, index=1))

  @qml.qnode(dev)
  def density_matrix_circuit():
      circuit()
      return qml.state()
  ```

  ```pycon
  >>> shots_circuit(shots=5)
  (array([0, 0, 0, 0, 0]), 0.19999999999999996)
  >>> density_matrix_circuit()
  tensor([[0.99750208+0.j, 0.04991671+0.j, 0.        +0.j],
         [0.04991671+0.j, 0.00249792+0.j, 0.        +0.j],
         [0.        +0.j, 0.        +0.j, 0.        +0.j]], requires_grad=True)
  ```

  However, there's one crucial ingredient that we still need to add: support for qutrit noise
  operations. Keep your eyes peeled for this to arrive in the coming releases!

> 
> <h3>Improvements 🛠</h3>

Reviewer: ```suggestion
```

<h4>Simulate mixed-state qutrit systems 3️⃣</h4>

> * Mixed qutrit states can now be simulated with the `DefaultQutritMixed` 
>   device (shortname: `"default.qutrit.mixed"`).

Reviewer: ```suggestion
* Mixed qutrit states can now be simulated with the `"default.qutrit.mixed" device.
```
Most users won't interact with the `DefaultQutritMixed` class but instead load it through `qml.device`, so probably makes sense to just mention that.

  [(#5495)](https://github.com/PennyLaneAI/pennylane/pull/5495)
  [(#5451)](https://github.com/PennyLaneAI/pennylane/pull/5451)
  [(#5186)](https://github.com/PennyLaneAI/pennylane/pull/5186)
  [(#5082)](https://github.com/PennyLaneAI/pennylane/pull/5082)
  [(#5213)](https://github.com/PennyLaneAI/pennylane/pull/5213)

>   Thanks to external contributors from the University of British Columbia, mixed 

Reviewer: ```suggestion
  Thanks to contributors from the University of British Columbia, a mixed-state
```

>   qutrit circuits are now available to program, having access to the same 
>   operations, measurements, and functionality as circuits programmed on 
>   `DefaultQutrit`.

Reviewer: ```suggestion
  qutrit device is now available for simulation, providing a noise-capable
  equivalent to `DefaultQutrit`.
```
Not sure If we should promise everything being the same

  ```python

>   dev = qml.device("default.qutrit.mixed", wires=1)

Reviewer: ```suggestion
  dev = qml.device("default.qutrit.mixed")
```

  def circuit():
      qml.TRY(0.1, wires=0)

  @qml.qnode(dev)
  def shots_circuit():
      circuit()
      return qml.sample(), qml.expval(qml.GellMann(wires=0, index=1))

  @qml.qnode(dev)
  def density_matrix_circuit():
      circuit()
      return qml.state()
  ```

  ```pycon
  >>> shots_circuit(shots=5)
  (array([0, 0, 0, 0, 0]), 0.19999999999999996)
  >>> density_matrix_circuit()
  tensor([[0.99750208+0.j, 0.04991671+0.j, 0.        +0.j],

>         [0.04991671+0.j, 0.00249792+0.j, 0.        +0.j],
>         [0.        +0.j, 0.        +0.j, 0.        +0.j]], requires_grad=True)

Reviewer: ```suggestion
         [0.04991671+0.j, 0.00249792+0.j, 0.        +0.j],
         [0.        +0.j, 0.        +0.j, 0.        +0.j]], requires_grad=True)
```
Do these need one more space worth of indent?

  ```

>   Keep your eyes peeled for qutrit noise model support in the coming releases!

Reviewer: ```suggestion
  However, there's one crucial ingredient that we still need to add: support for qutrit noise
  operations. Keep your eyes peeled for this to arrive in the coming releases!
```
Just want to address that we know it's a big piece that's still missing, but that we're on it.
